src/qp_klp/Workflows.py

```diff
@@ -517,8 +517,9 @@ def _compare_samples_against_qiita(self):
                 # check for possible match w/tube-ids, if defined in project.
                 results_tid = self._process_tube_ids(qiita_id, samples)
                 if results_tid:
-                    msgs.append("Number of values in sheet that aren't "
-                                "tube-ids in Qiita: %s" % len(results_tid[0]))
+                    rtid = results_tid[0]
+                    msgs.append('Number of sample-names not in Qiita: '
```

Comment:
I don't think this revised message is providing all the necessary info, which is that not only are these identifiers not found as sample names in qiita, they are also not found as tube ids in qiita.  I definitely would prefer that it not be identical in wording to the message at 507.  Can you modify?

Reply:
Yes!